src/program/snabb_lwaftr/Makefile

```diff
@@ -6,30 +6,30 @@ E= @echo
 #E= @:
 
 snabb-lwaftr:
-	$(Q) (cd ../../; make snabb-lwaftr; ln -sf ../../$@ program/snabb_lwaftr/$@)
```

Comment:
In `Makefile`s, command lists for targets must be indented with real tabs, because that's what marks the line as a command. Please do not replace tabs with spaces in this case.